lib/src/firestore.dart

```diff
@@ -4,6 +4,7 @@
 import 'dart:async';
 import 'dart:js';
 
+import 'package:firebase_admin_interop/js.dart';
```

Comment:
Curious why is this needed here? It only exports `bindings.dart` which is already imported here under `js` prefix.

Reply:
ok removed